tasks/less.js

```diff
@@ -146,6 +146,8 @@ module.exports = function(grunt) {
       });
     }
 
+    options.filename = path.resolve(options.filename);
```

Comment:
This looks like a bug in LESS, as it should support relative paths. Can you open an issue there?